tests/utils_tests/files/strip_tags2.txt

```diff
@@ -9,7 +9,7 @@ Before understanding what Fragments are and how they work, we must first realize
 
 The Problem
 -----------
-Suppose we have an Android app with two Activities- [*FirstActivity*](https://github.com/iontech/Fragments_Example/blob/master/src/main/java/com/github/iontech/fragments_example/FirstActivity.java) and [*SecondActivity*](https://github.com/iontech/Fragments_Example/blob/master/src/main/java/com/github/iontech/fragments_example/SecondActivity.java).
+Suppose we have an Android app with two Activities- [*FirstActivity*](https://github.com/iontech/Fragments_Example/blob/main/src/main/java/com/github/iontech/fragments_example/FirstActivity.java) and [*SecondActivity*](https://github.com/iontech/Fragments_Example/blob/main/src/main/java/com/github/iontech/fragments_example/SecondActivity.java).
```

Comment:
This repo doesn't exist (any more). We could perhaps rewrite this file to just exercise all syntax, rather than be a random markdown file

Reply:
Does it matter? It looks as though `strip_tags1.html` is also some random HTML file.
Either way, it's a separate issue. And broken links here don't affect anything anyway.

Reply:
Yeah it doesn't really matter, just a bit weird. Separate issue for sure.